src/Client.cpp

```diff
@@ -321,9 +324,17 @@ bool Client::OnQuery(
     CEF_REQUIRE_UI_THREAD();
 
     if (frame->IsMain() && m_remoteTrigger) {
-        m_jobManager->Process(browser, 200);
-        callback->Success("OK");
-        return true;
+	if (request == "trigger") {
+            callback->Success("Processing");
```

Comment:
I think this should also check that the `--remote-trigger` option is set before accepting the `trigger` request.

At this stands, a page could call `window.cefPdf({request: "trigger"})` to bypass `window.cefpdf.trigger` and produce a PDF even when the flag isn't set.